src/paymentservice/index.js

```diff
@@ -31,13 +28,13 @@ function chargeServiceHandler(call, callback) {
     span.setAttributes({
       'app.payment.amount': parseFloat(`${amount.units}.${amount.nanos}`)
     })
-    logger.info(`PaymentService#Charge invoked by: ${JSON.stringify(call.request)}`)
+    logger.info({request: call.request}, "Charge request received.")
 
     const response = charge.charge(call.request)
     callback(null, response)
 
   } catch (err) {
-    logger.warn(err)
+    logger.warn({err})
```

Comment:
Pino applies their error serialization to Error objects named `err` out of the box 👍 

- https://github.com/pinojs/pino-std-serializers#exportserrerror
- https://getpino.io/#/docs/api?id=serializers-object